dataforge-distributed/src/jvmMain/kotlin/space/kscience/dataforge/distributed/serialization/MetaCoder.kt

```diff
@@ -0,0 +1,23 @@
+package space.kscience.dataforge.distributed.serialization
+
+import io.lambdarpc.coding.Coder
+import io.lambdarpc.coding.CodingContext
+import io.lambdarpc.transport.grpc.Entity
+import io.lambdarpc.transport.serialization.Entity
+import io.lambdarpc.transport.serialization.RawData
+import kotlinx.serialization.json.Json
+import space.kscience.dataforge.meta.Meta
+import space.kscience.dataforge.meta.MetaSerializer
+import java.nio.charset.Charset
+
+internal object MetaCoder : Coder<Meta> {
```

Comment:
Is it possible to replace manual 'Coder's with KSerializer or IOFormat? Or make converters?

Reply:
Yes, it already exist, I just forgot about it.